Resources/Prototypes/Catalog/VendingMachines/Inventories/clothesmate.yml

```diff
@@ -42,6 +40,18 @@
     ClothingUniformJumpskirtColorDarkGreen: 2
     ClothingUniformJumpsuitColorTeal: 2
     ClothingUniformJumpskirtColorTeal: 2
+    ClothingUniformJumpsuitColorPurple: 2
+    ClothingUniformJumpskirtColorPurple: 2
```

Comment:
Any reason for moving these down a few lines?

Reply:
nope, was a mistake, I'll correct it